react-native-libraries.json

```
@@ -9300,5 +9300,14 @@
     "npmPkg": "@sparkfabrik/react-native-idfa-aaid",
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/matinzd/react-native-health-connect",
+    "npmPkg": "react-native-health-connect",
+    "nameOverride": "React Native Health Connect",
+    "examples": ["https://github.com/matinzd/react-native-health-connect/tree/main/example"],
+    "ios": false,
+    "android": true,
+    "newArchitecture": true
```

```suggestion
    "githubUrl": "https://github.com/matinzd/react-native-health-connect",
    "nameOverride": "React Native Health Connect",
    "examples": ["https://github.com/matinzd/react-native-health-connect/tree/main/example"],
    "android": true
```

Hello @matinzd, thank you for the new addition! 👍 

I have made few tweaks to the entry, for `newArchitecure` the autodetection will work in your package case, since you have `codegenConfig` filed defined in your `package.json`, so it can be safely omitted, same with `npmPkg` which is redundant in the case when repository name matches package name.

---

Nice!